packages/@lightningjs/ui-components/src/components/Provider/Provider.js

```diff
@@ -67,12 +67,17 @@ export default class Provider extends Base {
     const providerList = [];
     this.providers.slice(0, this.visibleCount).forEach(provider => {
       // Create a starting point for each provider icon.
+
       let patch = {
         centerInParent: true,
         radius: this.disableRadius ? 0 : this.style.radius,
-        alpha: this.style.alpha,
-        style: provider.style
+        alpha: this.style.alpha
       };
+
+      if (provider.style) {
+        patch.style = provider.style;
+      }
```

Comment:
does this matter? if style is set to undefined, does anything actually happen? Or could we keep it inline and do provider.style || {}?